Copy_Files.bat

```diff
@@ -0,0 +1,18 @@
+echo on
```

Comment:
Wouldn't just adding these files to the project in Visual Studio allow you to remove the triplicate copy of Copy_Files?

Reply:
I already did that. There's no copy_files.bat in current version. What you're looking at is some new code merged with all the reverts back to some ancient fronter version.

Reply:
Solved ✅